docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -282,4 +284,19 @@ Name of the vault to access. Sub-domain of `vault.azure.net`.
 
 ```bash tab="Example"
 --keyvault-name="MyKeyVault" 
-```
\ No newline at end of file
+```
+
+## Multifile Options
+
+### directory
+
+Path to the directory containing the keys and password files.
```

Comment:
keys -> key 

Suggested change so it's parallel with password

Reply:
Done.